src/components/snackbar/snackbar.tsx

```diff
@@ -174,61 +172,46 @@ export class Snackbar {
         }
     }
 
-    private renderActions(actionText: string, dismissible: boolean) {
-        if (!actionText && !dismissible) {
+    private renderDismissButton(dismissible: boolean) {
+        if (!dismissible) {
             return;
         }
 
+        const label = translate.get('snackbar.dismiss', this.language);
+
         return (
-            <div class="mdc-snackbar__actions" aria-atomic="true">
-                {this.renderActionButton(actionText)}
-                {this.renderDismissButton(dismissible)}
+            <div class="dismiss">
+                <limel-icon
+                    class="close mdc-snackbar__dismiss"
+                    name="multiply"
+                    title={label}
+                />
             </div>
         );
     }
 
-    private renderActionButton(actionText: string) {
+    private renderActions(actionText: string) {
         if (!actionText) {
             return;
         }
 
         return (
-            <limel-button
-                class="mdc-button mdc-snackbar__action"
-                label={actionText}
-            />
+            <div class="mdc-snackbar__actions" aria-atomic="true">
+                {this.renderActionButton(actionText)}
+            </div>
         );
     }
 
-    private renderDismissButton(dismissible: boolean) {
-        if (!dismissible) {
+    private renderActionButton(actionText: string) {
+        if (!actionText) {
             return;
         }
 
-        const label = translate.get('snackbar.dismiss', this.language);
-
         return (
-            <div class="dismiss">
-                {this.renderTimeoutVisualization()}
-                <limel-icon-button
-                    class="mdc-icon-button mdc-snackbar__dismiss"
-                    icon="multiply"
-                    label={label}
-                />
-            </div>
-        );
-    }
-
-    private renderTimeoutVisualization() {
-        return (
-            <svg width="36" height="36" viewBox="0 0 36 36">
-                <circle r="18" cx="18" cy="18" fill="var(--track-color)" />
-                <path
-                    class="track"
-                    d="M 18,18 m -16,0 a 16,16 0 1,0 32,0 a 16,16 0 1,0 -32,0"
-                    stroke="var(--fill-color)"
-                />
-            </svg>
```

Comment:
it seems you removed the countdown timer too. was this causing the confusion you mean?

Reply:
yes, exactly.